src/main/java/cz/mzk/fedora/solrcorrection/SolrVcRecords.java

```diff
@@ -0,0 +1,68 @@
+package cz.mzk.fedora.solrcorrection;
+
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.SolrServerException;
+import org.apache.solr.common.SolrDocument;
+import org.apache.solr.common.SolrInputDocument;
+
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class SolrVcRecords {
```

Comment:
Trida ma strasne divny nazev, ktery rika nepravdu o tom, co uvnitr dela. Kdyz prectu `SolrVcRecords` tak bych osobne ocekaval, ze je to seznam Solr zaznamu, podobne jak to ma `SolrDocumentList`. Ale tady to uvnitr dela veskerou logiku, to by fakt chtelo aspon jiny nazev.

Reply:
S pomenovaním tried, metód,... je to dosť obtiažne :D

Reply:
Máš pravdu, tohle se mi občas také nejde dobře :D ale tomu se naučiš s čásem